examples/example_bulkwriter.py

```diff
@@ -41,65 +44,119 @@
+        FieldSchema(name="float", dtype=DataType.FLOAT),
+        FieldSchema(name="double", dtype=DataType.DOUBLE),
+        FieldSchema(name="varchar", dtype=DataType.VARCHAR, max_length=512),
+        FieldSchema(name="json", dtype=DataType.JSON),
+        FieldSchema(name="vector", dtype=DataType.BINARY_VECTOR, dim=DIM) if bin_vec else FieldSchema(name="vector", dtype=DataType.FLOAT_VECTOR, dim=DIM),
+    ]
+    schema = CollectionSchema(fields=fields, enable_dynamic_field=True)
+    return schema
+
+def read_sample_data(file_path: str, writer: [LocalBulkWriter, RemoteBulkWriter]):
+    csv_data = pd.read_csv(file_path)
+    print(f"The csv file has {csv_data.shape[0]} rows")
+    for i in range(csv_data.shape[0]):
+        row = {}
+        for col in csv_data.columns.values:
+            if col == "vector":
+                vec = json.loads(csv_data[col][i])
+                row[col] = vec
+            else:
+                row[col] = csv_data[col][i]
+
+        writer.append_row(row)
+
 def test_local_writer_json(schema: CollectionSchema):
-    local_writer = LocalBulkWriter(schema=schema,
-                                   local_path="/tmp/bulk_data",
-                                   segment_size=4*1024*1024,
-                                   file_type=BulkFileType.JSON_RB,
-                                   )
-    for i in range(10):
-        local_writer.append_row({"id": i, "vector": [random.random() for _ in range(DIM)], "desc": f"description_{i}"})
+    print(f"\n===================== test local JSON writer ====================")
+    with LocalBulkWriter(
+            schema=schema,
+            local_path="/tmp/bulk_writer",
+            segment_size=4*1024*1024,
+            file_type=BulkFileType.JSON_RB,
+    ) as local_writer:
+        read_sample_data("./data/train_embeddings.csv", local_writer)
+        local_writer.commit()
+        batch_files = local_writer.batch_files
 
-    local_writer.commit()
-    print("test local writer done!")
-    print(local_writer.data_path)
-    return local_writer.data_path
+    print(f"Test local writer done! output local files: {batch_files}")
 
-def test_local_writer_npy(schema: CollectionSchema):
-    local_writer = LocalBulkWriter(schema=schema, local_path="/tmp/bulk_data", segment_size=4*1024*1024)
-    for i in range(10000):
-        local_writer.append_row({"id": i, "vector": [random.random() for _ in range(DIM)], "desc": f"description_{i}"})
 
-    local_writer.commit()
-    print("test local writer done!")
-    print(local_writer.data_path)
-    return local_writer.data_path
+def test_local_writer_npy(schema: CollectionSchema):
+    print(f"\n===================== test local npy writer ====================")
+    with LocalBulkWriter(
+            schema=schema,
+            local_path="/tmp/bulk_writer",
+            segment_size=4*1024*1024,
+    ) as local_writer:
+        read_sample_data("./data/train_embeddings.csv", local_writer)
```

Comment:
Is it possible that csv is too big(100 GB) that cannot load into the processing?